addons/mail/static/src/components/attachment_card/attachment_card.xml

```diff
@@ -45,13 +45,13 @@
                             <!-- Remove button -->
                             <t t-if="attachmentCard.attachment.isEditable">
                                 <button class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemUnlink btn top-0 justify-content-center align-items-center d-flex w-100 h-100 rounded-0" t-attf-class="{{ attachmentCard.attachmentList.composerViewOwner ? 'o-pretty position-absolute btn-primary' : 'bg-300' }}" t-on-click="attachmentCard.onClickUnlink" title="Remove">
-                                    <i class="fa fa-trash"/>
+                                    <i class="fa fa-trash" role="img" aria-label="Remove"/>
                                 </button>
                             </t>
                             <!-- Download button -->
                             <t t-if="!attachmentCard.attachmentList.composerViewOwner and !attachmentCard.attachment.isUploading">
                                 <button class="o_AttachmentCard_asideItem o_AttachmentCard_asideItemDownload btn d-flex justify-content-center align-items-center w-100 h-100 rounded-0 bg-300" t-on-click="attachmentCard.attachment.onClickDownload" title="Download">
-                                    <i class="fa fa-download"/>
+                                    <i class="fa fa-download" role="img" aria-lable="download"/>
```

Comment:
typo `aria-lable`, and `download` should probably start with upper case

Reply:
Fixed, thanks